examples/step-60/doc/results.dox

```diff
@@ -125,6 +125,88 @@ subsection Distributed Lagrange<1,2>
     set Variable names      = x,y,t
   end
 
+  subsection Embedding Dirichlet boundary conditions
```

Comment:
The documentation/commentary for these new `set` functions is exactly the same as what's found elsewhere in the tutorials, right?

Reply:
Yes

Reply:
Great, thanks for the confirmation. I did a quick check and it looked identical, but I thought I'd just make sure.

Reply:
Ah, I see that you did this already. Awesome :-)